.travis.yml

```diff
@@ -43,18 +43,23 @@ jobs:
   - stage: test
     env:
     - BUNDLE_VERSION=1.12.0
-    - GEMSETS="test sidekiq coverage"
+    - GEMSETS="test sidekiq"
     rvm: 2.3.0
   - stage: test
     env:
     - BUNDLE_VERSION=1.12.0
     - GEMSETS="test sidekiq"
     rvm: 2.4.1
+  - stage: test
+    env:
+    - BUNDLE_VERSION=1.12.0
+    - GEMSETS="test sidekiq coverage"
+    rvm: 2.5.1
```

Comment:
I think we should use 2.5 as we don't want to have to bump for patches. Similar for the other versions.

Reply:
I've updated to use patch independent versioning in all cases except for `1.9.3`.